test/parallel/test-buffer-readuintle.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUIntLE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUIntLE(2);
+
+assert.strictEqual(result, 168);
+
+assert.throws(
+      () => { buf.readUIntLE(5);}, RangeError);
```

Comment:
This the type of thing that could and should be enforced via linting. So let's do that: https://github.com/nodejs/node/pull/10377